Junchao-Mellanox/sonic-buildimage
# Add make support for thermal control feature

**- What I did**
1. Enable auto start for thermalctld in pmon dokcer
2. Add make support for  thermalctld 

**- How I did it**

**- How to verify it**
Build and install SONiC, reboot and go to pmon dokcer and see if there is a  thermalctld damon alive.

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
